src/app/page.js

```diff
@@ -7,7 +7,7 @@ export default function Home() {
   const cookieStore = cookies();
   const authSession = cookieStore.get('authsession');
   return (
-    <div className="margin-x-3">
+    <div>
```

Comment:
If we don't want an extra `div` in our markup this can also just be `<>` and the closing tag would be `</>` - these are for react only and don't get output to the DOM.

Reply:
Oops good point, I'll get rid of it!